packages/vm/src/compatability.rs

```diff
@@ -185,12 +186,69 @@ mod test {
         }
     }
 
-    // TODO: test check_wasm_memories
+    #[test]
+    fn test_check_wasm_memories_ok() {
```

Comment:
Can you add an explicit check with the test contract?

I guess this is tested with `check_wasm`, but nice to ensure we don't block our reference contracts.

Reply:
This is indeed covered by

```rust
    fn test_check_wasm() {
        // this is our reference check, must pass
        check_wasm(CONTRACT).unwrap();
    }
```

and I'd like to avoid adding test that don't test explicitely one thing. The case with `(memory 1)` is the same as our contract (which uses `(memory 17)`)